benchmark/benchmark.py

```diff
@@ -63,22 +63,30 @@ def _mount_mp(
 
     Returns Mountpoint version string.
     """
+    bucket = cfg['s3_bucket']
+    stub_mode = str(cfg["stub_mode"]).lower()
 
     if cfg['mountpoint_binary'] is None:
         mountpoint_args = [
             "cargo",
             "run",
             "--quiet",
             "--release",
-            "--",
+            "--features=mock",
         ]
+
+        if stub_mode == "s3_client":
+            logging.debug("using mock-mount-s3 due to `stub_mode`, bucket will be prefixed with \"sthree-\"")
```

Comment:
Why? Just to make sure we're not actually touching s3? (should be included as a comment)